docs/UsersGuide/best/rate-group.md

```diff
@@ -1,7 +1,46 @@
-## Rate Groups and Timeliness
+# Rate Groups and Timeliness
 
-A rate group is a container of *run()* ports that calls ports in order.
-The list of run ports does not know which destinations are in active
-components; however, the rate group is an active component.
+Often embedded software must perform actions at a fixed rate. In a given system there are usually collections of actions
+that must run at similar rates. For example, control algorithms may run at 10Hz while telemetry collection may run at
+1Hz and background tasks may be updated at 0.1Hz. F´ provides a mechanism to trigger time based actions called "rate
+groups". A rate group contains multiple output `Sched` ports that it sends a messages to at a repeated rate. Thus
+components having an input `Sched` port can run a repeated action at this rate. Rate groups are driven by a central rate
```

Comment:
Optional: Does it make sense to use the correct component names to reduce confusion? I.E. Active Rate Groups are driven by central rate group drive